resources.html

```diff
@@ -33,7 +33,7 @@ <h1 class="pagetitle">Resources</h1>
   <h2 class="subheading">Courses</h2>
     <ol>
       <li><a href="http://javascript.info" target="blank">The Modern JavaScript Tutorial</a></li>
-        <p>How it's done now. From the basics to advanced topics with simple, but detailed explanations.</p>
```

Comment:
Why are you removing the p's?

Reply:
while I was adding the information on the sites I remembered early on in the class about using figures and figure attributions for the redwood image. I wanted to make sure that any description I referenced was properly attributed to the site it came from so I figured I'd try to use that tag and see how it looked.

also when I played with the formatting using <figure> instead of <p>, I liked the way it automatically indented the content.

so.. personal preference plus paranoia?